tests/test_units/test_metadata_class.py

```diff
@@ -324,39 +321,35 @@ def test_metadata_access_deprecated_input(globalconfig1, regsurf):
     assert mymeta["access"]["classification"] == "restricted"
 
 
-def test_metadata_access_illegal_input(globalconfig1, regsurf):
+def test_metadata_access_illegal_input(globalconfig1):
     """Test giving illegal input, should provide empty access field"""
 
     # Input is "secret"
-    with pytest.warns(UserWarning):
-        edata = dio.ExportData(
+    with pytest.raises(ValueError, match="is not a valid AccessLevel"):
+        dio.ExportData(
             config=globalconfig1,
             access_ssdl={"access_level": "secret"},
             content="depth",
         )
-    assert not edata._config_is_valid
-
-    with pytest.raises(ValidationError, match="ssdl.access_level"):
-        generate_export_metadata(regsurf, edata)
 
     # Input is "open". Not allowed, shall fail.
-    with pytest.warns(UserWarning):
-        edata = dio.ExportData(
+    with pytest.raises(ValueError, match="is not a valid AccessLevel"):
+        dio.ExportData(
             config=globalconfig1,
             access_ssdl={"access_level": "open"},
             content="depth",
         )
-    assert not edata._config_is_valid
-    with pytest.raises(ValidationError, match="ssdl.access_level"):
-        generate_export_metadata(regsurf, edata)
 
 
 def test_metadata_access_no_input(globalconfig1, regsurf):
     """Test not giving any input arguments."""
 
+    # test assumption, deprected access.ssdl.access_level not present in config
+    assert "access_level" not in globalconfig1["access"]["ssdl"]
+
     # No input, revert to config
     configcopy = deepcopy(globalconfig1)
-    configcopy["access"]["ssdl"]["access_level"] = "restricted"
+    configcopy["access"]["classification"] = "restricted"
     configcopy["access"]["ssdl"]["rep_include"] = True
```

Comment:
Did we discuss taking `rep_include` out of config?

Reply:
yes, we discussed.. and to me it makes sense to take it out. But I figured since most people have it there now, we could allow it for a while. I'll add a separate issue to deprecate it 👍

Reply:
OK sounds good. Will we trigger warning now if it is missing from the config? If not, I guess we can hint to people that it _can_ be removed already following this PR.